README.md

```diff
@@ -1 +1,136 @@
-# budget_app
\ No newline at end of file
+# Budget App
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+    - [🚀 Live Demo](#live-demo)
+    - [🎬 Video Description](#video-demo)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Budget App <a name="about-project"></a>
+
+**Budget App** is a ruby on rails mobile web application that enables users to manage their budgets. The app provides a list of transactions created by a user associated with a category so that they can see how much they spent and on what.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+
+### Key Features <a name="key-features"></a>
+
+- **[Authenticated signup/signin]**
+- **[Add new category]**
+- **[Make a transaction]**
+- **[View all transactions and total expenditure]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+Click on the link to view the app.
+
+- [Live Demo](https://budget-tracker-lkkh.onrender.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] When I try to test the deployed version of your app, an error is raised when I go to sign in or sign-up pages, Please fix that. As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=The%20project%20should%20be%20deployed%20and%20accessible%20online.) your app supposed to be functioning and accessible.

Reply:
The live demo link has now been fixed.